src/platform/plugins/shared/data/server/search/strategies/common/async_utils.ts

```diff
@@ -43,7 +43,8 @@ export function getCommonDefaultAsyncSubmitParams(
     // If search sessions are used, store and get an async ID even for short running requests.
     keep_on_completion: useSearchSessions,
     // The initial keepalive is as defined in defaultExpiration if search sessions are used or 1m otherwise.
-    keep_alive: keepAlive,
+    // just for debugging purposes!
+    keep_alive: `60m`,
```

Comment:
ok, this is what we need to test it until ES resolves ignoring `keep_alive` in the ESQL case. Until this is resolved, it is necessary to set a manual values, preventing to have the session cleaned up after a minute